src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java

```diff
@@ -11,16 +12,41 @@
   private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
 
   @Test(
-          groups = "AdsUapDesktop"
+          dataProviderClass = AdsDataProvider.class,
+          dataProvider = "adsVUAPDesktop",
+          groups = "AdsVuapFlow"
   )
-  public void testVideoPlayFlow() {
-    Page page = new Page("project43", "SyntheticTests/VUAP");
+  public void testVideoPlayFlow(String slotName, Page page) {
     AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForPage(page), DESKTOP_SIZE);
 
-    VUAP vuap = new VUAP(driver, "TOP_LEADERBOARD");
+    VUAP vuap = new VUAP(driver, slotName);
     vuap.play();
 
     vuap.waitForVideoPlayerVisible();
     vuap.waitForVideoPlayerHidden();
   }
+
+  @Test(
+          dataProviderClass = AdsDataProvider.class,
+          dataProvider = "adsVUAPDesktop",
+          groups = "AdsVuapSizes"
+  )
+  public void testSlotSizes(String slotName, Page page) {
+    final int IMAGE_WIDTH = 1830;
+    final int IMAGE_HEIGHT = 744;
+
+    final int VIDEO_WIDTH = 1830;
+    final int VIDEO_HEIGHT = 1034;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00117)